src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,193 @@
+import json
+from _collections import OrderedDict
+
+import click
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.date_helper import parse_max_timestamp
+from code42cli.date_helper import parse_min_timestamp
+from code42cli.logger import get_logger_for_server
+from code42cli.options import begin_option
+from code42cli.options import end_option
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.options import send_to_format_options
+from code42cli.options import server_options
+from code42cli.output_formats import OutputFormatter
+from code42cli.util import warn_interrupt
+
+EVENT_KEY = "events"
+AUDIT_LOGS_KEYWORD = "audit-logs"
+
+AUDIT_LOGS_DEFAULT_HEADER = OrderedDict()
+AUDIT_LOGS_DEFAULT_HEADER["timestamp"] = "Timestamp"
+AUDIT_LOGS_DEFAULT_HEADER["type"] = "Type"
+AUDIT_LOGS_DEFAULT_HEADER["actorName"] = "ActorName"
+AUDIT_LOGS_DEFAULT_HEADER["actorIpAddress"] = "ActorIpAddress"
+AUDIT_LOGS_DEFAULT_HEADER["success"] = "Success"
+AUDIT_LOGS_DEFAULT_HEADER["resultCount"] = "ResultCount"
```

Comment:
Can we add affected username to the table output list? I think anything that we can filter for in the cmd options should be visible in the table, otherwise you can get stuff like this: 

```
# code42 audit-logs search --affected-username "tim.abramson+support@code42.com" -b 90d
Timestamp                  Type   ActorName   ActorIpAddress   Success   ResultCount   
2020-11-19T03:16:18.239Z   None   None        None             None      None          

# code42 audit-logs search --affected-username "tim.abramson+support@code42.com" -b 90d -f JSON
{
    "type$": "audit_log::local_auth_only_changed/1",
    "timestamp": "2020-11-19T03:16:18.239Z",
    "userId": "942564422882759874",
    "userName": "tim.abramson+support@code42.com",
    "newValue": false
}
```

And I see that there are two "type" fields in the results `type$` and `type`, but we're only showing one in the table, but pretty much all events that are not of type 'query' don't have the `type` field. And from the looks of it, Console is using the `type$` field to pull out the event type to display, so I wonder if all new events will only have `type$` going forward, and if we should just switch to displaying that instead of `type`.

Reply:
Yeah, it also looks like when we use `--event-type` to filter, it only works on the `type$` values, so we should definitely switch to displaying that.

Reply:
Changed, since we can't accommodate all columns in table format, removed `resultCount` and `success` columns and added the requested one.